Cargo.toml

```diff
@@ -21,7 +21,7 @@ assert_matches = "1.5.0"
 base64 = "0.21.3"
 bitvec = { version = "1.0.1", features = ["serde"] }
 # Point to the latest commit of branch msl/snos-0.6.0-rc.2
-blockifier = { git = "https://github.com/Moonsong-Labs/blockifier", rev = "30bb3d29b695ec4e0e9e6dccb62737f8f9474a54", features = ["testing"] }
+blockifier = { git = "https://github.com/Moonsong-Labs/blockifier", rev = "98d764a270c6a1b280edb94c973171027e8611fc", features = ["testing"] }
```

Comment:
Where did you find this revision? Latest on `msl/snos-0.6.0-rc.2` is [983e3b1cdb6621c5e6fa600f47cd69bf9e287621](https://github.com/Moonsong-Labs/blockifier/commits/msl/snos-0.6.0-rc.2/).

Reply:
You are right! I put the hash commit of my PR, but forgot to update it after the squash.

https://github.com/Moonsong-Labs/blockifier/pull/4/commits/98d764a270c6a1b280edb94c973171027e8611fc

Fixed!